interface/ceed.c

```diff
@@ -570,7 +570,7 @@ int CeedSetDeterministic(Ceed ceed, bool is_deterministic) {
 
   @ref Backend
 **/
-int CeedSetBackendFunction(Ceed ceed, const char *type, void *object, const char *func_name, int (*f)()) {
+int CeedSetBackendFunctionImpl(Ceed ceed, const char *type, void *object, const char *func_name, void (*f)(void)) {
```

Comment:
I'm a bit confused as to why this is a `void (*f)(void)` instead of `int (*f)(void)` - shouldn't all our internal backend functions return an error code?

Reply:
GCC warns when casting function pointers to/from anything other than `void (*)(void)`. Inside the actual definition you can see that the `void (*)(void)` generic type is cast to `int (*)(void)` for `fpointer`.